Week3/homework/1-step3.js

```diff
@@ -2,6 +2,7 @@
 
 function foo(func) {
   // What to do here? 
+  return func 
 }
```

Comment:
Not quite.  All you will do here is return the function object.  You want to call `func` and return the value from calling `func`.
```
....
return func();
....
````